packages/vanilla/src/components/global-nav/top-nav/help/help.html

```diff
@@ -1,3 +1,3 @@
-<div class="hig__global-nav__help hig__global-nav__top-nav__item">
```

Comment:
Tests were failing because of competing specificity with `.hig__global-nav__top-nav__item`. This help item is already contained in a `.hig__global-nav__top-nav__item`, so it seems safe to remove this class.